### PR TITLE
Add PostgreSQL module

### PR DESCRIPTION
**Pull request by chrisvire**

### CODE REVIEW

File: aws/rds/postgresql/vars.tf [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  type = "string"
+}
+
+variable "subnet_group_name" {
+  type = "string"
+}
+
+variable "availability_zone" {
+  type    = "string"
+  default = ""
```
Comment (forevermatt) [inline aws/rds/postgresql/vars.tf]: If you want this to be a required variable, you probably don't want to provide a `default` here.

**Comment by forevermatt** [comment]
@chrisvire It looks like the only difference between the `mariadb` module and this new `postgresql` module is the default value for a variable. Would it work to simply use the existing module and provide an `engine` value of `'postgres'`?

**Comment by chrisvire** [comment]
@forevermatt, Another difference is the resource definition:
```resource "aws_db_instance" "pg_instance" {```

We have 2 databases (one MariaDB and one PostgreSQL).  I asked @fillup if these needed to be unique and he said that they did.  I am fine with whatever works...

**Comment by chrisvire** [comment]
I talked with Phillip and I should just use the mariadb and change the engine value (this is what GTIS is doing for postgresql databases).

**Pull request closed without merging**